cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -168,6 +168,19 @@ endif()
 
 _ov_find_dependency(Threads)
 
+set(ENABLE_INTEL_GNA "@ENABLE_INTEL_GNA@")
+if(${ENABLE_INTEL_GNA} AND NOT libGNA_FOUND)
```

Comment:
1. `${ENABLE_INTEL_GNA}` => `ENABLE_INTEL_GNA`
2. We need to enable this code only for static libraries case

Reply:
done